tests/fixtures/integration.py

```diff
@@ -0,0 +1,49 @@
+"""Pytest fixtures for integration tests."""
+
+from datetime import datetime
+from pathlib import Path
+
+import pooch
+import pytest
+
+GIN_TEST_DATA_REPO = "https://gin.g-node.org/SainsburyWellcomeCentre/crabs-exploration-test-data"
+
+
+@pytest.fixture(scope="session")
+def pooch_registry() -> dict:
+    """Pooch registry for the test data.
+
+    This fixture is common for all the test session. The
+    file registry is downloaded fresh for every test session.
+
+    Returns
+    -------
+    dict
+        URL and hash of the GIN repository with the test data
+
+    """
+    # Initialise pooch registry
+    registry = pooch.create(
+        Path.home() / ".crabs-exploration-test-data",
+        base_url=f"{GIN_TEST_DATA_REPO}/raw/master/test_data",
+    )
+
+    # Download the registry file from GIN
+    # Force to download it fresh every time by using a timestamped filename
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Comment:
as @niksirbi made me realise, if knonw_hash = None, the file is [always downloaded](https://github.com/fatiando/pooch/blob/78cf76046b7893d757930df4dffc0f7a44bac2c0/pooch/core.py#L241).

So there is no need to set it to a different name